addons/mail/views/mail_activity_views.xml

```diff
@@ -306,6 +307,7 @@
                 <field name="date_done" string="Done Date" optional="hide"/>
                 <button name="action_done" type="object" string="Done" icon="fa-check"/>
                 <button name="unlink" type="object" string="Cancel" icon="fa-times" class="text-danger"/>
+                <button name="action_snooze" class="text-warning" type="object" string="Snooze 7d" icon="fa-bell-slash"/>
```

Comment:
Probably not related to this task but... (Feel free to ignore): Does it makes sens to display the snooze button on activities that are not overdue?

I feel like the "snooze" feature should work like the snooze feature of the alarm clocks :alarm_clock: You should be able to click on the snooze button only after you've been waken up (i.e: when you're late).

Reply:
Hi @julien-banken :slightly_smiling_face: 
I feel like it might also be useful when the activity is planned for today. You have to do something today but you don't have time, it has been delayed or it can wait so you snooze it for 1 week :smile: What do you think?
For future activities it is a bit unlogical I agree :thinking: 
Is your remark also concerning the crm leads or only the activities?

(I'll send a message to lna to see what he thinks about it :smile:)

Reply:
For ref: lna wants to keep the snooze for every activities no matter the deadline
=> Snooze here is mostly used as an easy way to say "reschedule for later".